features/COMENTAR_EM_POST.feature

```diff
@@ -0,0 +1,51 @@
+Comentar em post
+
+Cenários da feature:
+
+Scenario: Postar comentário
+	Given  estou logado como "usuário comum” com login “ptlh” e senha “4321”
+	And    estou na página “Histórico de reviews” do usuário "agsa"
+	And    vejo o post do aluno “agsa” sobre o professor "professor A"
+	When   preenche a barra de comentários com “Muito legal!”
+	And    ainda estou na página “Histórico de reviews” do aluno "agsa"
+	Then   vejo a baixo do post do aluno “agsa” sobre o professor "professor A" o comentário “Muito legal!”
```

Comment:
Era para as palavras reservadas (Given, And, When...) estarem com highlight. Acredito que seja problema ou com as linhas de texto soltas ou a ideação que não está correta.

Reply:
Rapaz, verifiquei essas duas coisas e não tá ficando.

Reply:
Consertei.